### PR TITLE
On hover show timestamp on human readable timestamps (#2570)

### PR DESCRIPTION
**Pull request by mikaelz**
1. At http://calypso.localhost:3000/posts
2. Hover human readable timestamp and should see machine timestamp

ss_2017-07-25-07-12-52

### CODE REVIEW

**Comment by designsimply** [comment]
Thanks for submitting a PR @mikaelz! I have added the `[Status] Needs Review` label which means a Calypso developer should review the code without needing to ping someone specifically. It would be very helpful as well if you could include a short list of steps for testing and possibly a screenshot for PRs like this one when you can.

**Comment by mikaelz** [comment]
1. At http://calypso.localhost:3000/posts
2. Hover human readable timestamp and should see machine timestamp

ss_2017-07-25-07-12-52

File: client/my-sites/post-relative-time-status/index.jsx [hunk 1]
```diff
@@ -47,9 +47,9 @@ module.exports = React.createClass( {
 		return (
 			<span className="post-relative-time-status__time">
 				<Gridicon icon="time" size={ 18 } />
-				<span className="post-relative-time-status__time-text">
+                <time className="post-relative-time-status__time-text" dateTime={ time } title={ time }>
```
Comment (aduth) [inline client/my-sites/post-relative-time-status/index.jsx]: There is whitespace inconsistency here. The Calypso code base uses tabs, not spaces.

**Comment by mikaelz** [comment]
Thanks @aduth changed.

**Comment by nb** [comment]
@mikaelz, thanks for the fix – I love those nice touches – I am one of those people who hover on times :-) Are there, by any chance, more place in Calypso where we would need a similar fix?

**Comment by aduth** [comment]
@nb See also #16861, which was a continuation of this, plus feedback at https://github.com/Automattic/wp-calypso/pull/16861#pullrequestreview-54739180 which mentions that this could be applied to other screens as well (currently only posts list).

**Pull request merged**